apps/jenkins/src/main/fabric8/openshift-deployment.yml

```diff
@@ -76,6 +76,8 @@ spec:
             secretKeyRef:
               name: jenkins-recommender-api-token
               key: token
+        - name: JENKINS_ROOT_URL
```

Comment:
Add value to Tempalte https://github.com/fabric8-services/fabric8-tenant-jenkins/blob/master/apps/jenkins/src/main/fabric8/template.yml